cluster/charts/crossplane-controllers/values.yaml.tmpl

```diff
@@ -21,4 +21,7 @@ hostedConfig:
   tenantKubernetesServiceHost: ""
   tenantKubernetesServicePort: ""
   controllerNamespace: ""
-  crossplaneSATokenSecret: ""
\ No newline at end of file
+  crossplaneSATokenSecret: ""
+
+templateStacks:
```

Comment:
want to also add these new config values to the table in https://github.com/crossplaneio/crossplane/blob/master/docs/install-crossplane.md#configuration?

Reply:
Yeah definitely - thanks for the link! I was not aware of this document o: